openedx/features/content_tagging/rules.py

```diff
@@ -53,8 +84,9 @@ def can_change_taxonomy(user: User, taxonomy: oel_tagging.Taxonomy = None) -> bo
     """
     if taxonomy:
         taxonomy = taxonomy.cast()
-    return is_taxonomy_admin(user, taxonomy) and (
-        not taxonomy or (taxonomy and not taxonomy.system_defined)
+
+    return not taxonomy or (
+        is_taxonomy_admin(user, taxonomy) and (not taxonomy.system_defined)
```

Comment:
Everyone can call PUT/PATCH/DELETE (taxonomy=None). The actual object permission (taxonomy!=None) will be handled in the second call.

Reply:
Done!